xml/art_admin_slemicro_selinux.xml

```diff
@@ -0,0 +1,353 @@
+  <para>
+   The policy is the key component in SELinux. Your SELinux policy defines
+   rules that specify which objects can access which files, directories, ports,
+   and processes on a system. To do this, a security context is defined for all
+   of these.
+  </para>
+  <para>
+   An SELinux policy contains a huge number of rules. To make it more
+   manageable, policies are often split into modules. This allows administrator
+   to switch protection on or off for different parts of the system.
+  </para>
+  <para>
+   When compiling the policy for your system, you will have a choice to either
+   work with a modular policy, or a monolithic policy, where one huge policy is
+   used to protect everything on your system. It is strongly recommended to use
+   a modular policy and not a monolithic policy. Modular policies are much
+   easier to manage.
+  </para>
+  <para>
+    &slema; si shipped with the <literal>targeted</literal> SELinux policy.
+  </para>
+ </section>
+ <section xml:id="sec-security-context">
+  <title>SELinux security context</title>
+  <para>
+   The security context is a set of information assigned to a file or a
+   process. It consists of identity, role, type and level. This informaton is
+   used to make access control decisions.
+  </para>
+  <variablelist>
+   <title></title>
+   <varlistentry>
+    <term>identity</term>
+    <listitem>
+     <para>
+      is an identity defined in the policy that is authorized for a specific
+      set of roles and for a specific <emphasis>level</emphasis> range. Each
+      Linux user is mapped to an SELinux user. SELinux does not use the list of
+      user accounts maintained by Linux in <filename>/etc/passwd</filename>,
+      but uses its own database and mapping. By convention, the identity name
+      is suffixed with <literal>_u</literal>, for example
+      <literal>user_u</literal>.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>role</term>
+    <listitem>
+     <para>
+      defines a set of permissions a user can be granted. A role defines which
+      <emphasis>types</emphasis> a user assigned this role can access. By
+      convention, the role name is suffixed with <literal>_r</literal>, for
+      example: <literal>system_r</literal>.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>type</term>
+    <listitem>
+     <para>
+      conveys information how particular files and processes can interact. A
+      process is assigned files with a concrete SELinux type and it cannot
+      acces files outside of this type. By convention, the type name is
+      suffixed with <literal>_t</literal>, for example:
+      <literal>var_t</literal>.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>level</term>
+    <listitem>
+     <para>
+      is an optional attribute that specifies the range of levels of clearance
+      in the multilevel security.
```

Comment:
a security context usually has five fields:
unconfined_u:unconfined_r:unconfined_t:s0-s0:c0.c1023
level is described here, but categories are missing